### PR TITLE
Add elpigraph-python to conda-forge

### PR DESCRIPTION
**Pull request by j-bac**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by auranic** [comment]
I agree to be a maintainer

File: recipes/elpigraph-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  description: |
+    elpigraph-python is a package implementing the Elastic Principal Graphs algorithm for manifold learning.
+  doc_url: https://elpigraph-python.readthedocs.io/
+  dev_url: https://github.com/elpigraph-python/elpigraph/
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```
Comment (scopatz) [inline recipes/elpigraph-python/meta.yaml:58]: Please remove comments

File: recipes/elpigraph-python/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - matplotlib
+  run:
+    - python
+    - numpy >=1.16.2
+    - pandas >=0.23.4
+    - numba >=0.49.1 
+    - scikit-learn >=0.21.3
+    - scipy >=1.2.0
+    - python-igraph >=0.7.1
+    - matplotlib
```
Comment (scopatz) [inline recipes/elpigraph-python/meta.yaml:35]: This should be `matplotlib-base`

File: recipes/elpigraph-python/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  host:
+    - python
+    - pip
+    - numpy >=1.16.2
+    - pandas >=0.23.4
+    - numba >=0.49.1 
+    - scikit-learn >=0.21.3
+    - scipy >=1.2.0
+    - python-igraph >=0.7.1
+    - matplotlib
```
Comment (scopatz) [inline recipes/elpigraph-python/meta.yaml:26]: Are all of these needed to build this package. This is usually just pip & python

File: recipes/elpigraph-python/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - scikit-learn >=0.21.3
+    - scipy >=1.2.0
+    - python-igraph >=0.7.1
+    - matplotlib-base
+
+
+test:
+  imports:
+    - elpigraph
+    - elpigraph.src
```
Comment (marcelotrevisani) [inline recipes/elpigraph-python/meta.yaml:34]: Can we just add ``pip check`` here please?
```suggestion
    - elpigraph.src
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/elpigraph-python/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15ac2dcefe4d2a0b475359d10471f0ec2e8c069100dc81208c924efe845231dd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+requirements:
```
Comment (marcelotrevisani) [inline recipes/elpigraph-python/meta.yaml:16]:
```suggestion

requirements:
```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**